### DISCUSSION TITLE
What software ?

### DISCUSSION THREAD

**Discussion by tonycstech** [kachurovskiy/nanoels#69] [Q&A]
Is there a free software that i can use to modify given PCB files ?
Need larger monitor and buttons spaced out further so i can use larger buttons.

**Comment by kachurovskiy** [comment]
Tony, PCB files can be edited in EasyEDA.

I've actually have a new hardware version for this project that has big screen, bigger buttons and some more improvements - https://github.com/kachurovskiy/nanoels/tree/main/h2

![image](https://user-images.githubusercontent.com/517919/188195916-fe182020-db9c-40cd-959e-4c15204dcc96.png)

I'm still working on the code but the PCB and case are usable. Let me know if you're ready to be the early adopter :) Cheers!

**Comment by tonycstech** [comment]
I just spent nearly $200 getting components for the 1st build.
I dont know how much more money i will need to build your latest version.
You had a joystick that i dont see in this  picture, not sure if you
abandoned it or what ?

I will try to build the older one and if i succeed, i will try newer
version.

**Reply by kachurovskiy** [reply]
Tony, this is the NanoEls project which is single-axis. If you're building MegaEls which is 2-axis then there was a mistake :)

**Reply by zabu83** [reply]
The h2 version of nanoels will support 2 axis?

**Reply by kachurovskiy** [reply]
@zabu83 no, NanoEls H2 is still single-axis

**Comment by tonycstech** [comment]
Yes. I want to have 2 axis.
Which one do i need ?
Please send me a link.
Thank you Максим.

**Reply by kachurovskiy** [reply]
I would recommend buying Rocketronics ELS 4 Basic.

If you're set on building one yourself, see https://github.com/kachurovskiy/megaels but prepare for a long journey :)

**Comment by tonycstech** [comment]
I bought everything I need for this version and printed all the parts.
Motor does not fit anywhere.

**Reply by kachurovskiy** [reply]
Hi @tonycstech, can you please post a picture showing your problem? Thank you
